README.md

```diff
@@ -1,13 +1,229 @@
 # CHINESE MENU GENERATOR-BACKEND
 
-## ERD
+#### Team Building API Exercise Using NodeJS/ExpressJS
+
+<hr style="border: solid 1px"/>
+
+## Table of Content
+
+- [Summary](#summary)
+- [Authors](#authors)
+- [Goals](#goals)
+  - [Project First Itearation Goals](#project-first-iteration-goals)
+  - [Extension Goals for Future Iterations](#extension-goals-for-future-iterations)
+- [Stack](#stack)
+- [Install Locally](#install-locally)
+- [Schema](#schema)
+- [Swagger](#swagger)
+- [Endpoints](#endpoints)
+  - [Postman](#postman)
+  - [User Endpoints](#user-endpoints)
+  - [Menu Endpoints](#menu-endpoints)
+  - [Recipe Endpoints](#recipe-endpoints)
+
+<hr style="border: solid 1px"/>
+
+## Summary
+
+The main goal of this exercise was to build an API that in NodeJS and ExpressJS that uses TypeScript including tests - Jest - and documentation - swagger.
```

Comment:
Can we remove the first "that" in line 28?

Reply:
✅